layers/amqp/Dockerfile

```diff
@@ -19,7 +21,7 @@ RUN cmake --build . --target install
 WORKDIR ${AMQP_BUILD_DIR}
 RUN git clone https://github.com/php-amqp/php-amqp
 WORKDIR ${AMQP_BUILD_DIR}/php-amqp
-RUN git reset --hard 618e06ad2ef867598831cdd3faadba0dd65be917
+RUN git checkout v2.1.2
```

Comment:
Could you explain this change?

Reply:
This commit referenced php-amqp/php-amqp@618e06a, a fix for PHP 8.1. I assume this was needed back then when this commit was not tagged yet. Tag v2.1.2 is from Jan 22, 2024 and this fix is already contained in it. So this is basically an update of php-amqp source to latest release.